recipes/zernike/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "zernike" %}
+{% set version = "0.0.32" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/zernike-{{ version }}.tar.gz
+  sha256: 00120a8552301b1d82161d9e86ef7c08cb275bea09798437c5be7728f5d28073
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - numpy
+    - h5py
```

Comment:
Unless the package imports itself at build time you don't need these here.

Reply:
I tried without it and the build failed 

https://github.com/conda-forge/staged-recipes/commit/b76a83005a0038eb0e365055f42245ae4b4a1af7

Reply:
The setup.py does some imports that are a bit unusual

Reply:
No problem. Just wanted to be sure. I'd open an issue upstream though.